src/server.ts

```diff
@@ -1,11 +1,12 @@
 import App from '@/app';
 import AuthRoute from '@routes/auth.route';
+import GroupsRoute from './routes/group.route';
```

Comment:
Follow the alias convention
```suggestion
import GroupsRoute from '@/routes/group.route';
```
Shouldn't this be plural too? 
groups.route ?